site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,548 @@
+you can still use semantic referencing across them!
+
+A simple reference is defined as such::
+
+    .. _reference-target-name::
+
+    This is a bit of content.
+
+    This is how you point to the above reference, :ref:`reference-target-name`
+
+Sphinx also includes a number of other semantic reference types.
+Here are a few examples:
+
+* ``:doc:`` for referencing documents
+* ``:cls:`` for referencing programming classes
+* ``:term:`` for referencing glosary terms
+
+All references support ``intersphinx``,
+which allows you to prefix your references with a specific project name.
+So if your user guide needs to reference your API documentation,
+you could do::
+
+    Check out the :cls:`api-ref:api.request`
+
+Which would know to look in your ``api-ref`` project for the class.
+The name ``api-ref`` is arbitrary,
+and is defined in your project configuration's ``intersphinx_mapping``.
+
+You see the complete set of references in the Sphinx documentation:
+
+* Cross Referencing Syntax - http://www.sphinx-doc.org/en/stable/markup/inline.html#cross-referencing-syntax
+* Intersphinx - http://www.sphinx-doc.org/en/stable/ext/intersphinx.html
+
+Including comments form source code
+```````````````````````````````````
+Integration with software is one of the largest benefits of Sphinx.
+You can easily embed software comments from multiple languages,
+including Python, Java, and .Net.
+
+Here is an example of embedding Python documentation for a class in your project::
+
+    Make a request
+    --------------
+
+    You can make requests using the :func:`api.request` module.
+    It makes HTTP requests against our website.
+    Here is the basic function signature:
+
+    .. autofunction:: api.request
+
+As you can see,
+you can include prose content along with generated content in the same file.
+This allows for building a narrative around generated code comments,
+instead of giving your users a separate User Guide and API Reference,
+which is often times just a alphabetical listing of code!
+
+Documentation is best written by humans.
+Pulling comments from source code is valuable,
+but it should be a small part of a proper set of documentation.
+
+You can read more about including code comments in the following documentation pages:
+
+* Domains, where the references are defined - http://www.sphinx-doc.org/en/stable/domains.html#what-is-a-domain
+* Autodoc, which generates docs from Python code - http://www.sphinx-doc.org/en/stable/ext/autodoc.html
+* Breathe, which bridges Doxygen and Sphinx - https://breathe.readthedocs.io/en/latest/
+* AutoAPI, which bridges .Net and Sphinx - https://sphinx-autoapi.readthedocs.io/en/latest/
+
+Tables
+------
+
+Working with tables can be the bane of anyone who uses plaintext markup languages.
+Most other languages require that you write them in the file with some arcane syntax.
+However with reStructuredText,
+you can use directives to make this much easier.
+
+It comes with two powerful list directives,
```

Comment:
"You can use either of two powerful list directives, ..." (Antecedent of "It" not clear ...)